.gitlab/source_test/ebpf.yml

```diff
@@ -53,29 +82,28 @@ tests_ebpf_arm64:
     when: always
     paths:
       - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files
-      - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files
```

Comment:
Just for double check, can you confirm this was not used and is not required on the `prepare_sysprobe_ebpf_functional_tests` above?
Thanks

Reply:
Yes it is not required since system-probe tests have been moved off of kitchen with [this PR](https://github.com/DataDog/datadog-agent/pull/24755).
This PR adds KMT specific tasks for setting up test packages instead of reusing old tasks used with the kitchen based system.